test/cmdlineTests.sh

```diff
@@ -225,6 +225,11 @@ EOF
         sed -i.bak -e 's/ Consider adding "pragma .*$//' "$stderr_path"
         sed -i.bak -e 's/\(Unimplemented feature error.* in \).*$/\1<FILENAME REMOVED>/' "$stderr_path"
         sed -i.bak -e 's/"version":[ ]*"[^"]*"/"version": "<VERSION REMOVED>"/' "$stdout_path"
+    if [[ $stdout_expectation_file != "" &&  $stderr_expectation_file != "" ]]
+    then
+        sed -i.bak -e '/^Compiler run successful. No contracts to compile\.$/d' "$stdout_path"
+        sed -i.bak -e '/^Compiler run successful. No output generated\.$/d' "$stdout_path"
```

Comment:
Ah wait, this is still not properly escaped:
```suggestion
        sed -i.bak -e '/^Compiler run successful\. No contracts to compile\.$/d' "$stdout_path"
        sed -i.bak -e '/^Compiler run successful\. No output generated\.$/d' "$stdout_path"
```

Reply:
Done.